Test/TestConfigurationUtils.ps1

```diff
@@ -205,15 +205,46 @@ function Test-IfVmSwitchExist {
         [Parameter(Mandatory = $true)] [String] $VmSwitchName
     )
 
+
+    Write-Log "Checking if VmSwitch '$VMSwitchName' exists..."
+
     $r = Invoke-Command -Session $Session -ScriptBlock {
         Get-VMSwitch $Using:VMSwitchName -ErrorAction SilentlyContinue | Select-Object -ExpandProperty isDeleted
     }
 
-    if(($null -eq $r) -or ($r.Equals($true))) {
+    if($null -eq $r) {
+        Write-Log '... it does not.'
         return $false
     }
+    elseif($r.Equals($true)) {
+        Write-Log '... it still exist, but is deleting.'
+        return $false
+    }
+    elseif($r.Equals($false)) {
```

Comment:
Why not just `else`?

Reply:
Just to be sure that it returned false not just some mambo jumbo.